server/services/publication-service.js

```diff
@@ -1,20 +1,17 @@
-'use strict';
-
-const { getPluginService } = require('../utils/getPluginService');
-const { getPluginEntityUid } = require('../utils/getEntityUId');
-const { getDeepPopulate } = require('../utils/populate');
+import getPluginService from '../utils/getPluginService';
+import getPluginEntityUid from '../utils/getEntityUId';
+import getDeepPopulate from '../utils/populate';
 
 const actionUId = getPluginEntityUid('action');
-
-module.exports = ({ strapi }) => ({
+export default ({ strapi }) => ({
 	/**
 	 * Publish a single record
 	 *
 	 */
-	async publish(uid, entityId, data = {}) {
+	async publish(uid, entityId = {}) {
 		const populateRelations = strapi.config.get('server.webhooks.populateRelations', true);
```

Comment:
This seems to have been removed https://www.strapi.cn/dev-docs/migration/v4-to-v5/breaking-changes/remove-webhook-populate-relations

Reply:
Context in Publisher Plugin: https://github.com/pluginpal/strapi-plugin-publisher/issues/51